roles/firmware/tasks/main.yml

```diff
@@ -18,7 +18,7 @@
 
 - name: Install firmware (for RPi internal WiFi)
   include_tasks: install.yml
-  #when: firmware_installed is undefined
+  when: firmware_installed is undefined
```

Comment:
Debatable.

Downloading the latest firmware every time might in fact be desired and preferable.

Reply:
Both iiab-network and admin-console's 'services' but network goes along for the ride, are expected to work offline or when the internet is not available. Not sure what effect of SKIP_ROLE_ON_ERROR might have changed things.